specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/common.json

```diff
@@ -0,0 +1,245 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Cognitive Language Service",
+    "description": "The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>.",
+    "version": "2021-11-01-preview"
+  },
+  "paths": {},
+  "definitions": {
+    "ErrorResponse": {
+      "type": "object",
+      "description": "Error response.",
+      "additionalProperties": false,
+      "properties": {
+        "error": {
+          "description": "The error object.",
+          "$ref": "#/definitions/Error"
+        }
```

Comment:
Based on feedback to 2022-03-01-preview, `error` should be required.

Reply:
good catch. I made it required.